form/dynamic_form_modification.rst

```diff
@@ -32,8 +32,8 @@ take a look at the :doc:`Form Events </form/events>` documentation.
 Customizing your Form Based on the Underlying Data
 --------------------------------------------------
 
-Before jumping right into dynamic form generation, hold on and recall what
```

Comment:
I agree on the tone which is slightly different here compared to the rest of the documentation.
If `hold on` is removed I think `jumping right into` should also be changed as it relates to each other.

Also the double semicolon (::) should be kept before a code example.